test/integration-config.json

```diff
@@ -16,6 +16,10 @@
     "CONNECT_TIMEOUT": 45000,
     "RECONNECT_INTERVAL": 5000
   },
+  "DB_CONNECTION": {
+    "POOL_MIN": 10,
+    "POOL_MAX": 30
```

Comment:
Same as above.

Reply:
Have a question regarding this - initial request was to remove the prefix. Can we please verify?